src/containers/errors/error-500.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+  import { H1 } from 'attractions';
+  export let error;
+
+  const dev = process.env.NODE_ENV === 'development';
+</script>
+
+<div class="material padded">
+  {#if dev}
+    <H1>Alright, so here's what happened</H1>
+    {#if error.stack}
+      <pre>{error.stack}</pre>
+    {/if}
+  {:else}
+    <H1>Everything is fine</H1>
```

Comment:
Reminds me of this 😂 
![image](https://user-images.githubusercontent.com/11016151/85954985-338e3200-b984-11ea-85ab-1392aaa992c2.png)
Actually, how about we add it there 😃?

Reply:
Oh, darn, it reads "this is fine". This is exactly what I was referring to, but if you want to, we can add a picture too :p

Reply:
I think it would be funny)